### ISSUE TITLE
typoes in Versions.inc

### ISSUE THREAD

**Issue by Koooper** [Slluxx/Picklock_RCM#4]
For some strange reason, LP appears a few times in versions.inc??? shouldn't it be PL for picklock??? smh

**Comment by R3m0ved** [comment]
Another topic but still related, will overall renaming impact hekate 'Lockpick' payload integration ? (The one mapped under **Tools**)

**Comment by R3m0ved** [comment]
> > For some strange reason, LP appears a few times in versions.inc??? shouldn't it be PL for picklock??? smh
> 
> Yeah, i guess. Its hard to find all occurrences of a name abbreviation if you don't know what to look for. Its not like i would read each line meticulously. Ill go over it and try to change what i can find before the next keys have to be put in.
> 
> > Another topic but still related, will overall renaming impact Hekate 'Lockpick' payload integration ? (The one mapped under Tools)
> 
> I am not sure where you see Lockpick under tools in Hekate. Maybe you mean the "Payloads" section? That section is empty by default and you can place whatever payload you want in `/bootloader/payloads/` to show up in that menu.

My bad, it's under "Console info" section

![image](https://github.com/Slluxx/Picklock_RCM/assets/7079018/ce15a149-5c7f-4d73-b939-20c38eafa9ef)

I think hekate does or override something when lauching Lockpick from here, as you can directly reboot to hekate from Lockpick payload

**Comment by Slluxx** [comment]
Yes. The name change will interfere with that button. [Here is the line in Hekates code.](https://github.com/CTCaer/hekate/blob/937ab52d1498bfacc0f432e580bc12ffce000a7f/nyx/nyx_gui/frontend/gui_info.c#L2408). However Hekate does not override anything in lockpick. You can boot to Hekate from Picklock too.

Apart from that, the button is not really important as you can run any Payload from the Payloads section. Its up to CTCaer to change this if he wishes to but i don't think he will (at least for now) just because "some guy made a renamed fork of lockpick".
